frontend/src/components/RegistrationView/FormFields.tsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FieldProps, Field } from 'formik';
+import { get } from 'lodash';
+
+
```

Comment:
With this much css and different styled components, I'd split components in this file into a different files. Improves readability